include/seqan3/alphabet/composite/all.hpp

```diff
@@ -7,26 +7,26 @@
 
 /*!\file
  * \author Marcel Ehrhardt <marcel.ehrhardt AT fu-berlin.de>
- * \brief Meta-header for the composition submodule; includes all headers from alphabet/composition/.
+ * \brief Meta-header for the composite submodule; includes all headers from alphabet/composite/.
  */
 #pragma once
 
-#include <seqan3/alphabet/composition/cartesian_composition.hpp>
-#include <seqan3/alphabet/composition/union_composition.hpp>
+#include <seqan3/alphabet/composite/alphabet_tuple_base.hpp>
+#include <seqan3/alphabet/composite/alphabet_variant.hpp>
 
-/*!\defgroup composition Composition
+/*!\defgroup composite Composite
  * \brief Provides data structures joining multiple alphabets into a single alphabet.
  * \ingroup alphabet
  *
  * \par Introduction
  *
- * Composition alphabets are special alphabets that allow you to combine existing alphabets into new ones. For example,
- * you can add new characters to existing alphabets by using seqan3::union_composition or combine alphabets with quality
- * information by using seqan3::cartesian_composition.
+ * This module provides various templates that allow you to combine existing alphabets into new ones. For example,
```

Comment:
I wouldn't use the word `template` here as inexperienced users (in terms of metaprogramming) might be confused and not know that it just means "some class template".
Maybe `classes` or `helper classes` or at least `class templates`